sport/app/rugby/feed/OptaFeed.scala

```diff
@@ -0,0 +1,42 @@
+package rugby.feed
+
+import common.{ExecutionContexts, Logging}
+import play.api.Play.current
+import play.api.libs.ws.{WSRequest, WS}
+import rugby.jobs.RugbyStatsJob
+import rugby.model.LiveScore
+
+import scala.concurrent.Future
+
+case class RugbyOptaFeedException(message: String) extends RuntimeException(message)
+
+object OptaFeed extends ExecutionContexts with Logging {
+  private val xmlContentType = ("Accept", "application/xml")
+
+  private def getLiveScoresResponse: Future[String] = {
+
+    val endpoint = conf.Configuration.optaRugby.endpoint
+    endpoint.map { e =>
+      WS.url(e)
+        .withHeaders(xmlContentType)
+        .withQueryString("competition" -> "3", "season_id" -> "2016", "psw" -> conf.Configuration.optaRugby.apiKey.getOrElse(""),
```

Comment:
these magic strings would be great to have as `val` s for easy reading/tweaking.